src/models/Message.model.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+import { DataTypes } from 'sequelize';
+import { sequelize } from '../db.js';
+
+export const Message = sequelize.define(
+  'message',
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    roomId: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    text: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    time: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    author: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+  },
+  {
+    tableName: 'messages',
+    updatedAt: false,
+    createdAt: false,
+  },
+);
+
+Message.sync();
```

Comment:
Calling `Message.sync()` directly in the model file might not be the best practice. You usually want to control when and where your database synchronization happens. Consider moving this to a separate database setup or initialization file where you handle all your models' synchronization.